packages/web-app-files/tests/unit/components/AppBar/ViewOptions.spec.ts

```diff
@@ -33,9 +35,28 @@ describe('ViewOptions component', () => {
     wrapper.find(selectors.fileExtensionsSwitch).vm.$emit('change', false)
     expect(storeOptions.modules.Files.mutations.SET_FILE_EXTENSIONS_VISIBILITY).toHaveBeenCalled()
   })
+  it('initially does not show a viewmode switcher', () => {
+    const { wrapper } = getWrapper()
+    expect(wrapper.find('[data-testid="viewmode-switch-buttons"]').exists()).toBeFalsy()
+  })
+  it('shows a viewmode switcher if more than one viewModes are passed', () => {
+    const { wrapper } = getWrapper(
+      {},
+      {
+        viewModes: [ViewModeConstants.condensedTable, ViewModeConstants.default]
+      }
+    )
+    const viewModeSwitchButtons = wrapper.find('[data-testid="viewmode-switch-buttons"]')
+    expect(viewModeSwitchButtons).toMatchSnapshot()
```

Comment:
Would like to avoid a snapshot test for this. Should already be sufficient to check the presence of the element.
If you ever find a situation where a snapshot test is useful, please do so on the `.html()` result of the element. Will be required shortly.

Reply:
I'm somewhat torn about this - I know the oC web team in general is not a huge fan of snapshot tests, however in this case it's not only asserting that there is an element with a `viewmode-switch-buttons` data-testid, but also that there's two buttons, each with an icon inside (and checks for the *momentarily* correct CSS classes etc)

Reply:
Also fine to keep it... I'd go the way of asserting that there are `x` buttons and ignoring the icon assertion for example.

But: please change line 50 to `expect(viewModeSwitchButtons.html()).toMatchSnapshot()`